### ISSUE TITLE
@codepath

### ISSUE THREAD

**Issue by kyleplattner** [kyleplattner/dropbox#1]
@codepath

**Comment by drnick23** [comment]
This weeks goal was to achieve parity with prototyping frameworks by learning how to layout views, create hotspots, and navigate between screens using various animations.

It looks like you've had difficulty fully grasping a few concepts -- while you added a performSegueWithIdentifier in a view controller file, you did not name the file correctly (for instance, as part of the CreateNewAccountViewController) and map it to the view controller in the storyboard.

Your tab views could have included nested navigation views instead of overlaying custom headers yourself. The navigation controller is important under the tabs, because that's what allows you to have separate navigation histories for each tab.

Under normal circumstances I would say this was an incomplete assignment. However, I suspect you did not budget enough time, but it's not clear to me based on your work that you absorbed and applied enough material to keep up moving forward. Many of the items to be learned from this assignment will carry forward into the next assignment, so I would budget at least 10 hours to catch up if you want to reach your goals for the course. Please meet or email me before the lab session on Thursday to talk about whether you really want to continue in the course.

**Comment by kyleplattner** [comment]
You are correct that I didn’t budget enough time. I’d like to continue in the course. 

Kyle

**Comment by drnick23** [comment]
Hi Kyle,

Are you attending the lectures remotely? I don't think you were here for
yesterday's lecture (unless you didn't sign the attendance sheet).

Nick

**Comment by kyleplattner** [comment]
I don’t work out of the SF office so I’m remote.

**Comment by drnick23** [comment]
OK -- it can be tricky to absorb most of the material from just the
recorded lectures. In general, the assignments will be about 80% of the
learning experience, and the labs will contribute towards your
understanding of how to do various parts of the assignments too.

Please budget a little time getting started on your assignment before the
lab, as you'll have the most benefit from the lab at that point. Spending 2
hours following the lab steps will make the assignments feel more familiar.


Also -- it doesn't look like you've joined the Facebook group -- let me
know if you need a different email for the invite or you can request to
join at https://www.facebook.com/groups/865122760232351/868954743182486/

I know people have issues with using Facebook, but this is really just used
for support and many of the assignment's little quirks are answered in
there every week, so you are shooting yourself in the foot if you are
unable to read or post in there.
